### PR TITLE
Izzy Allum hw01

### PR DESCRIPTION
**Pull request by iallum**
I followed along with the video for the basic process of forking and cloning/downloading the hw01 repository (into my cfss folder). Then I wrote out a short bio for myself and googled how to add the elements asked for in the assignment. I found an R Markdown cheatsheet and used that as the general guideline for how to do things. I also found information from an RStudio article on Markdowns as well as online forums. I then saved, staged, committed, and pushed my code to GitHub. Next, I figured out (using my good friend Google again) how to preview the HTML and PDF versions of my Markdown to see how it would turn out/if I had done things correctly (I realize now that was unnecessary-my bio just wasn’t showing up like I wanted on GitHub because my code had errors). I then tinkered around with my code through a process of trial and error with the aid of online references until it was showing up like I wanted. The most difficult part was figuring out how to get my picture to show up (first in the previews and then on GitHub) since that required me learning that I had to upload my picture to a folder in R and that I couldn’t just draw it from my desktop. Throughout this process I saved my code every now and then to save my progress. At the end, close to when I was finished with my code, I had some problems getting my code to push to GitHub and I ended up with lots of different commits. But after asking for help on the GitHub discussion, Professor Soltoff told me how to fix the problem (I had to pull from GitHub) and also assisted me with my questions getting images to show. I then did a final save, stage, commit, and push to GitHub before doing a pull request. Tgere were some frustrating parts (when I got stuck and didn’t know what to do), but overall it was kind of fun.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
-# hw01
\ No newline at end of file
+### Self-Introduction  
+
+![](images/profile_pic.png)  
+
+Hi! My name is **Izzy Allum**, and I am a first year at the University of Chicago. I'm from Washington, DC, so that's where I've been spending quaratine. My hobbies include:
```
Comment (bensoltoff) [inline README.md:5]: Are you actually DC, or just the DMV? I'm originally from Loudoun County.

Also, I have no idea when the "DMV" became a thing.

File: README.md [hunk 2]
```diff
@@ -1 +1,10 @@
-# hw01
\ No newline at end of file
+### Self-Introduction  
+
+![](images/profile_pic.png)
```
Comment (bensoltoff) [inline README.md:3]: Nice addition of the image, plus a good idea to store it in a separate images folder. It pays to keep your repos organized so everything isn't scattered in the same main folder

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Reproducibility**         |       x   |              |            |

* Nice mix of Markdown elements

**Comment by rammkripa** [comment]
EDITED: I accidentally posted my review as a comment. My bad!

File: README.md (hunk 1 above)
Comment (iallum) [reply README.md:5]: actually DC!

File: README.md [hunk 3]
```diff
@@ -1 +1,10 @@
...
\ No newline at end of file
+### Self-Introduction  
+
+![](images/profile_pic.png)  
+
+Hi! My name is **Izzy Allum**, and I am a first year at the University of Chicago. I'm from Washington, DC, so that's where I've been spending quaratine. My hobbies include:  
+
+* fencing (I'm on the club team at UChicago)
+* baking and cooking
+* listening to [Spotify](https://open.spotify.com/) and watching [Netflix](https://www.netflix.com/browse)
```
Comment (YinsuH) [inline README.md:9]: This is a really good way for people to track the information you shared with a link attached! Nice job!

**Review by YinsuH — COMMENTED** [review]
Hi Izzy,
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   x        |              |            |
| **Coding strategy**         |   x        |              |            |
| **Presentation**            |   x        |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Reproducibility**         |      x     |              |            |

I really like your doc! Everything is clear and structured. And you have well-included everything required for this homework. I didn't know how to create a link under a specific name until I read your doc, so really thank you for that!

File: README.md [hunk 4]
```diff
@@ -1 +1,10 @@
-# hw01
\ No newline at end of file
+### Self-Introduction  
+
+![](images/profile_pic.png)  
+
+Hi! My name is **Izzy Allum**, and I am a first year at the University of Chicago. I'm from Washington, DC, so that's where I've been spending quaratine. My hobbies include:  
+
+* fencing (I'm on the club team at UChicago)
+* baking and cooking
```
Comment (rammkripa) [inline README.md:8]: Cooking is indeed amazing.
I highly recommend The Chef's Table on Netflix!

**Review by rammkripa — COMMENTED** [review]
Nice bio, Izzy!
I liked your explanation of the process you used, especially the parts that involved trail, error and google(super relatable!)
I also think your use of bold formatting, links, and lists was cool.
I am definitely borrowing your idea of having a separate images folder.
A small improvement might be adding a title to your image (by writing it in the square brackets).
Another minor detail you might want to change is one of your spellings. (crafsty-->craftsy)
Good luck for the rest of the course!

**Pull request closed without merging**